Docs entry for the `is_quantized`

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32075 Docs entry for the `is_quantized`**

Differential Revision: [D19353861](https://our.internmc.facebook.com/intern/diff/D19353861)

Commit:
Docs entry for the `is_quantized`

jerryzh168 reviewed (approved):
Thanks!

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit c04bbbf1:


* **2/2** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=c04bbbf14bbf61f0fb1514e84f93214d359adad3).

### :detective: 2 new failures recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4210776) pytorch_linux_xenial_py2_7_9_test (1/2)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=83597542) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=83597542))
<details>
<summary>
<code>Jan 10 23:04:18 RuntimeError: test_docs_coverage failed!</code>
</summary>

```
Jan 10 23:04:18  
Jan 10 23:04:18 ---------------------------------------------------------------------- 
Jan 10 23:04:18 Ran 2 tests in 0.004s 
Jan 10 23:04:18  
Jan 10 23:04:18 FAILED (failures=1) 
Jan 10 23:04:18 Traceback (most recent call last): 
Jan 10 23:04:18   File "test/run_test.py", line 456, in <module> 
Jan 10 23:04:18     main() 
Jan 10 23:04:18   File "test/run_test.py", line 449, in main 
Jan 10 23:04:18     raise RuntimeError(message) 
Jan 10 23:04:18 RuntimeError: test_docs_coverage failed! 
Jan 10 23:04:18 + cleanup 
Jan 10 23:04:18 + retcode=1 
Jan 10 23:04:18 + set +x 
Jan 10 23:04:18 =================== sccache compilation log =================== 
Jan 10 23:04:18 ERROR:sccache::server: Compilation failed: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "/tmp/torch_extensions/test_compilation_error_formatting/main.cpp: In function \'int main()\':\n/tmp/torch_extensions/test_compilation_error_formatting/main.cpp:2:23: error: expected \';\' before \'}\' token\n int main() { return 0 }\n                       ^\n" } 
Jan 10 23:04:18  
Jan 10 23:04:18 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 10 23:04:18 Compile requests                 46 
Jan 10 23:04:18 Compile requests executed        27 
Jan 10 23:04:18 Cache hits                       20 
```

</details>

#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4210779) pytorch_linux_xenial_py3_6_gcc5_4_test (2/2)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=83600142) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=83600142))
<details>
<summary>
<code>Jan 10 23:05:12 RuntimeError: test_docs_coverage failed!</code>
</summary>

```
Jan 10 23:05:12  
Jan 10 23:05:12 ---------------------------------------------------------------------- 
Jan 10 23:05:12 Ran 2 tests in 0.005s 
Jan 10 23:05:12  
Jan 10 23:05:12 FAILED (failures=1) 
Jan 10 23:05:12 Traceback (most recent call last): 
Jan 10 23:05:12   File "test/run_test.py", line 456, in <module> 
Jan 10 23:05:12     main() 
Jan 10 23:05:12   File "test/run_test.py", line 449, in main 
Jan 10 23:05:12     raise RuntimeError(message) 
Jan 10 23:05:12 RuntimeError: test_docs_coverage failed! 
Jan 10 23:05:12 =================== sccache compilation log =================== 
Jan 10 23:05:12 + cleanup 
Jan 10 23:05:12 + retcode=1 
Jan 10 23:05:12 + set +x 
Jan 10 23:05:12 ERROR:sccache::server: Compilation failed: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "/tmp/torch_extensions/test_compilation_error_formatting/main.cpp: In function \'int main()\':\n/tmp/torch_extensions/test_compilation_error_formatting/main.cpp:2:23: error: expected \';\' before \'}\' token\n int main() { return 0 }\n                       ^\n" } 
Jan 10 23:05:12  
Jan 10 23:05:12 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 10 23:05:12 Compile requests                 43 
Jan 10 23:05:12 Compile requests executed        25 
Jan 10 23:05:12 Cache hits                       18 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

Commit:
Update on "Docs entry for the `is_quantized`"


Differential Revision: [D19353861](https://our.internmc.facebook.com/intern/diff/D19353861)
